install_ffmpeg.sh

```diff
@@ -159,6 +160,6 @@ fi
 
 if [[ ! -e "$ROOT/ffmpeg/libavcodec/libavcodec.a" || $BUILD_TAGS == *"debug-video"* ]]; then
   cd "$ROOT/ffmpeg"
-  make $FFMPEG_MAKE_EXTRA_ARGS
-  make install
+  make -j$NPROC $FFMPEG_MAKE_EXTRA_ARGS
```

Comment:
Seems that we can remove `FFMPEG_MAKE_EXTRA_ARGS` here and in the definition earlier in the file as mentioned by @cyberj0g since all it does right now is set `-j4` which is no longer needed with this change.

Reply:
Done!